src/audible_cli/models.py

```diff
@@ -143,7 +143,7 @@ class LibraryItem(BaseItem):
     def _prepare_data(self, data: dict) -> dict:
         return data.get("item", data)
 
-    def _get_codec(self, quality: str):
+    def _get_codec(self, quality: str) -> tuple[str | None, int | None]:
```

Comment:
this is incorrect notation. without a lot of rejiggering, safest would just to have it noted as `-> tuple:`